official/vision/beta/projects/yt8m/utils.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2016 Google Inc. All Rights Reserved.

```

Comment:
Is this file only used by the dataloaders?
If that's the case, could you move it into the dataloaders/ folder?